.travis.yml

```diff
@@ -26,7 +28,9 @@ matrix:
     env: ECLIPSE_TARGET=neon
   - jdk: oraclejdk8
     env: ECLIPSE_TARGET=oxygen
-script: mvn -Ptravis --fail-at-end verify ${ECLIPSE_TARGET:+-Declipse.target=${ECLIPSE_TARGET}}
+script:
+  - mvn -Ptravis --fail-at-end verify ${ECLIPSE_TARGET:+-Declipse.target=${ECLIPSE_TARGET}}
+  - mvn cobertura:cobertura
```

Comment:
Ah!  The problem is that `verify` doesn't install the built results.  Since you have `cobertura:cobertura` call in a separate step, it doesn't have the previously-built artifacts in its reactor and so `cobertura` attempts to resolve the artifacts.

We could have Travis do a `mvn install` rather than a `mvn verify`, but then our built artifacts will be cached by Travis, which is not a good thing.

But combining these into one line should work.